src/components/transactions/UnStake/UnStakeActions.tsx

```diff
@@ -25,12 +25,12 @@ export const UnStakeActions = ({
   ...props
 }: UnStakeActionProps) => {
   const { currentAccount } = useWeb3Context();
-  const stakingService = useStakeTxBuilderContext(selectedToken);
+  const redeem = useRootStore((state) => state.redeem);
 
   const { action, loadingTxns, mainTxState, requiresApproval } = useTransactionHandler({
     tryPermit: false,
     handleGetTxns: async () => {
-      return stakingService.redeem(currentAccount, amountToUnStake.toString());
+      return redeem(selectedToken)(currentAccount, amountToUnStake.toString());
```

Comment:
@satanworker e.g. here the account is kinda unnecessary

Reply:
yep, removed